feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/utils.c

```diff
@@ -207,7 +207,14 @@ int phy_get_tx_chainmask(const char *name)
 		return 0;
 	return phy->tx_ant;
 }
+int phy_get_rx_chainmask(const char *name)
+{
+	struct wifi_phy *phy = phy_find(name);
 
+	if (!phy)
+		return 0;
+	return phy->rx_ant;
+}
 int phy_get_channels(const char *name, int *channel)
```

Comment:
The API to get the MAX available antennas from the phy record in radio_nl80211.c can go here.

Reply:
> The API to get the MAX available antennas from the phy record in radio_nl80211.c can go here.

Done